test/fixtures/hospital.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
Here's an example fixtures file, in json format, showing how you can create test data for the database.